### PR TITLE
feat: channel info drop dowm from the sidebar leading to browse channels and create channel

### PR DESCRIPTION
**Pull request by sulaimon23**
# Fixes Issue/Linear Ticket

> This could be an existing issue or a linear ticket 
> - Github Issue Example: My PR Closes #{ISSUE}
> - Linear Ticket Example: Fixes ID-#{ISSUE}

# Changes proposed

> Talk about the things you did eg. files changes, dependencies installed e.t.c 

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zurimain/blob/main/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zurimain/blob/main/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
Screenshot 2023-02-08 at 3 01 11 PM

- [x] I am making a pull request against the **main branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

### CODE REVIEW

**Comment by sulaimon23** [comment]
@ickynavigator

File: packages/ui/src/Layout/components/PluginItem.tsx [hunk 1]
```diff
@@ -162,9 +193,31 @@ const PluginItem: React.FC<PluginItemProps> = ({
...
+              gap="sm"
+              justify="flex-end"
+              align="center"
+              direction="row"
+              wrap="wrap"
+            >
+              <ChannelAction />
+            </Flex>
+          )}
+        </Flex>
```
Comment (ickynavigator) [inline packages/ui/src/Layout/components/PluginItem.tsx:220]: looks like this can be handled with either a tooltip or a menu component from mantine

**Review by ickynavigator — CHANGES_REQUESTED** [review]

File: packages/ui/src/Layout/components/ChannelAction.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    color: theme.colorScheme === "dark" ? theme.white : theme.black
+  },
+  wrapper: {
+    backgroundColor:
+      theme.colorScheme === "dark"
+        ? theme.colors.dark[7]
+        : theme.colors.secondary[4]
+  }
+}));
+//
```
[the remark this answers is not in the document]
Comment (sulaimon23) [reply packages/ui/src/Layout/components/ChannelAction.tsx:15]: Done

File: packages/ui/src/Layout/components/ChannelAction.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        </Menu.Target>
+        <Menu.Dropdown className={classes.wrapper}>
+          <Menu.Item>
+            <Text className={classes.text} size="xs">
+              Browse channels
+            </Text>
+          </Menu.Item>
+          <Menu.Item>
+            <Text className={classes.text} size="xs">
+              Create a channel
```
[the remark this answers is not in the document]
Comment (sulaimon23) [reply packages/ui/src/Layout/components/ChannelAction.tsx:36]: Yes.. they are meant to navigate to messaging

File: packages/ui/src/Layout/components/PluginItem.tsx (hunk 1 above)
Comment (sulaimon23) [reply packages/ui/src/Layout/components/PluginItem.tsx:220]: No,, its not a menu or accordion.. its a conditional PLus icon that shows on hover then you click and see browse channel and create channel..

**Comment by sulaimon23** [comment]
@ickynavigator

File: packages/ui/src/Layout/components/ChannelAction.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (ickynavigator) [reply packages/ui/src/Layout/components/ChannelAction.tsx:35]: alright

**Review by ickynavigator — APPROVED** [review]

**Pull request merged**